molgenis-app-gavin2/src/main/java/org/molgenis/app/WebAppInitializer.java

```diff
@@ -0,0 +1,14 @@
+package org.molgenis.app;
+
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+import org.molgenis.core.ui.MolgenisWebAppInitializer;
+import org.springframework.web.WebApplicationInitializer;
+
+public class WebAppInitializer extends MolgenisWebAppInitializer
+    implements WebApplicationInitializer {
+  @Override
+  public void onStartup(ServletContext servletContext) throws ServletException {
+    super.onStartup(servletContext, WebAppConfig.class, 150);
```

Comment:
This will make 150MB the maximum file upload size for this app (might be restricted further by the servlet container). Is this enough for the uploaded vcfs?

Reply:
According to Joeri:
- Typically file size will be around 1 or 2 MB
- Maximum around 100MB